zpl_generator.py

```
@@ -304,59 +304,41 @@ def generate_dry_label(printer_id):
     """
 
 
+def validate_serial_number(serial: str) -> bool:
+    """Validate serial number format and length for Code128 barcode."""
+    # Check format matches pattern: 5 digits-11 digits
+    if not re.match(r'^\d{5}-\d{11}$', serial):
```

Ruff (0.8.0)

Undefined name `re`

(F821)